### DISCUSSION TITLE
Is there a way to create a jwt token to be used on pytest?

### DISCUSSION THREAD

**Discussion by yolo3231** [fastapi-users/fastapi-users#810] [Q&A]
Hi, thanks for this fabulous application,
I have successfully made an API using fastapi users, it has been great.
I would like to implement some tests using pytest, the main idea is to generate the JWT token and reuse it as fixture scope between the other tests, but I can't find any example in the documentation.

I have the idea to choose a user from user manager and then generate the token using _generate token function
Something like this:

```

@pytest.fixture(scope="session")
async def session_token():
    async with get_user_db_context() as user_db:
        async with get_user_manager_context(user_db) as user_manager:
            user = await user_manager.get_by_email("test@example.com")
            jwt = await jwt_authentication._generate_token(user)
            return f'Bearer {jwt}'


  def test_(self, session_token):
        response = client.get("url",
                              headers={"'authorization'": session_token},
                              params={
                                 ...
                              })
        assert response.status_code == status.HTTP_200_OK

```

Thank you for that amazing support!

**Comment by EssaAlshammri** [comment]
[https://github.com/fastapi-users/fastapi-users/discussions/671#discussioncomment-2481105](here)
